src/compiler/binder.ts

```diff
@@ -3372,7 +3372,7 @@ namespace ts {
                 emitFlags |= NodeFlags.HasAsyncFunctions;
             }
 
-            if (currentFlow && isObjectLiteralOrClassExpressionMethod(node)) {

```

Comment:
There are only 3 callers of `isObjectLiteralOrClassExpressionMethod`. Two of them need this addition, and the third is guarded by `case SyntaxKind.MethodDeclaration`. So I'd like to see the new check moved into that function, and the name should be ... uhh ... `isObjectLiteralOrClassExpressionMethodOrAccessor` ???

Maybe there's a better name if there's a term that means "method or accessor".

Reply:
I just went with choice 1, `isObjectLiteralOrClassExpressionMethodOrAccessor`; there was also a `ContainerFlags` by the same name that looked like it could be part of the same renaming effort.